f8a_report/manifest_helper.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
         """Init method for the Report helper class."""
         self.s3 = S3Helper()
         self.curr_dir = os.path.join(
-            os.path.abspath(str(Path.home())), "f8a_report", "manifests")
+            '/tmp', "dynamic_manifests")
```

Comment:
have you checked this filepath works in your dev cluster?

Reply:
Yes, I have tested. 
![S3 Management Console](https://user-images.githubusercontent.com/21026016/68647997-4a8f2980-0545-11ea-8ceb-616851e955ec.jpg)